en/docs/install-and-setup/setup/security/configuring-transport-level-security.md

```diff
@@ -24,7 +24,7 @@ To prevent these types of security attacks, it is encouraged to disable the weak
      See the list of [supported cipher suites](https://docs.wso2.com/display/ADMIN44x/Supported+Cipher+Suites) .
 
 4.  Start the server.
-5.  To verify that the configurations are all set correctly, download and run the [TestSSLServer.jar]({{base_path}}/assets/attachments/administer/TestSSLServer.jar).
+5.  To verify that the configurations are all set correctly, download and run the [TestSSLServer.jar]({{base_path}}/assets/attachments/admin/TestSSLServer.jar).
```

Comment:
@samgnaniah this file names needs to be changed to lower case in order for the link to work. This file name has been references in other places in this page so best to change all instances that this is mentioned.